batch/batch/front_end/front_end.py

```diff
@@ -1132,7 +1140,12 @@ async def create_batch(request, userdata):
 
     batch_spec = await request.json()
     id = await _create_batch(batch_spec, userdata, db)
-    return web.json_response({'id': id})
+    n_jobs = batch_spec['n_jobs']
```

Comment:
I want to disallow empty updates entirely. I feel like we should just be able to reject API requests that try to add an update with 0 jobs in it, and the clients can make it easier by just making empty updates a no-op.

Reply:
That's a good approach. I think now that we think of batches as containers of jobs, this makes more sense. I think I was trying to lift the exact semantics that we allowed previously. I like your approach better especially since updates are not user visible.

The implication of this is that we do not allow empty job groups as well. I think that's fine, but wanted to make sure this was considered.

Reply:
Just to confirm, we need to clean this up in a future PR in the next PR stack to raise an exception?

Reply:
What here would raise an exception? In a future PR that introduces updates, I'll raise a BadRequest error if someone tries to submit an update with 0 jobs in it.

The current behavior which I am keeping here is that Batches can have 0 jobs, it will simply be a batch with no updates.